cli/commands/clean.js

```diff
@@ -117,6 +122,16 @@ exports.run = function (logger, config, cli) {
 	} else if (appc.fs.exists(buildDir)) {
 		logger.debug(__('Deleting all platform build directories'));
 
+		// scan platform SDK specific clean hooks
+		if (ti.targetPlatforms) {
+			ti.targetPlatforms.forEach(function(platform) {
+				var platformCleanModule = path.join(__dirname, '..', '..', platform, 'cli', 'commands', '_clean.js');
```

Comment:
same as above. I don't think we need the special _clean.js middle-man for loading hooks.